### PR TITLE
Fix/add documentation

### PR DESCRIPTION
**Pull request by ValentinMerlet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ Update your database schema to add our model
 ```
 bin/console doctrine:schema:update --force
 ```
+or if you use [DoctrineMigrationsBundle](https://github.com/doctrine/DoctrineMigrationsBundle) : 
+```
+bin/console doctrine:migrations:migrate
+```
```
Comment (Peekmo) [inline README.md:80]: Is that really necessary ? If the user have an extra bundle to manage its database, it's his problem in my opinion

**Review by artggd — APPROVED** [review]

**Comment by artggd** [comment]
LGTM!

File: README.md [hunk 2]
```diff
@@ -311,7 +315,7 @@ A CRUD is provided in order to administrate your menus. It's available on the ro
 
 /!\ You need to have the role `ROLE_TREE_MENU_ADMIN` in order to be able to access the route.
 
-Start by running `php bin/console assets:install` to get the assets in your web directory.
+Start by running `php bin/console assets:install` to get the assets in your web/public directory.
```
Comment (Peekmo) [inline README.md:318]: "web or public" instead "web/public" ?

File: README.md (hunk 1 above)
Comment (ValentinMerlet) [reply README.md:80]: DoctrineMigrationsBundle is integrated to the core in sf4

File: README.md (hunk 2 above)
Comment (ValentinMerlet) [reply README.md:318]: 'm gonna change this

**Pull request merged**